docs/user_guide/mitigation.rst

```diff
@@ -194,6 +207,684 @@ Postprocessing
+on the output of the decision function. Syntax-wise, this works exactly the
+same as specifying a threshold as in the previous examples. Note that
+if :code:`predict_method` is set to :code:`decision_function`, the default
+threshold will automatically be set to 0, as that is the value used by
+classifiers that predict using :code:`decision_function`.
+
+
+.. doctest:: mitigation
+    :options: +NORMALIZE_WHITESPACE
+
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from fairlearn.postprocessing import plot_histograms_per_group
+    
+    >>> classifier_log_reg = LogisticRegression().fit(X_train,Y_train) 
+    >>> Y_pred_decision_func = classifier_log_reg.decision_function(X_test)
+
+    >>> #Check distribution of decision function output
+    >>> plot_histograms_per_group(sf_test,Y_pred_decision_func)
+
+.. figure:: ../auto_examples/images/user_guide_thresholder_decision_func_distr.png
+    :align: center
+
+.. doctest:: mitigation
+    :options: +NORMALIZE_WHITESPACE
+
+    >>> #Check unthresholded predictions
+    >>> Y_pred = classifier_log_reg.predict(X_test)
+
+    >>> MetricFrame(metrics=false_negative_rate, 
+    ...             y_true=Y_test,
+    ...             y_pred=Y_pred, 
+    ...             sensitive_features=sf_test).by_group
+    race
+    AfricanAmerican    0.425676
+    Asian              0.487805
+    Caucasian          0.453915
+    Hispanic           0.426087
+    Other              0.437500
+    Name: false_negative_rate, dtype: float64
+
+    >>> plot_positive_predictions(sf_test,Y_pred)
+
+.. figure:: ../auto_examples/images/user_guide_thresholder_decision_func_positive_pred_no_threshold.png
+    :align: center
+
+.. doctest:: mitigation
+    :options: +NORMALIZE_WHITESPACE    
+
+    >>> #Specify threshold, and inspect the difference
+    >>> threshold_dict = {'Asian': -0.5}
+
+    >>> thresholder = Thresholder(estimator=classifier_log_reg,
+    ...                           threshold_dict=threshold_dict,
+    ...                           prefit=True,
+    ...                           predict_method='decision_function')\
+    ...                           .fit(X_train,Y_train,sensitive_features=sf_train)
+
+    >>> Y_pred_thresholded = thresholder.predict(X_test,sensitive_features=sf_test)
+
+    >>> MetricFrame(metrics=false_negative_rate, 
+    ...             y_true=Y_test,
+    ...             y_pred=Y_pred_thresholded, 
+    ...             sensitive_features=sf_test).by_group
+    race
+    AfricanAmerican    0.425676
+    Asian              0.048780
+    Caucasian          0.453915
+    Hispanic           0.426087
+    Other              0.437500
+    Name: false_negative_rate, dtype: float64
+
+    >>> plot_positive_predictions(sf_test,Y_pred_thresholded)
+
+.. figure:: ../auto_examples/images/user_guide_thresholder_decision_func_positive_pred_yes_threshold.png
+    :align: center    
+
+
+
+
+Specify threshold for multiple sensitive features
```

Comment:
I’d say we keep it in until we have the functionality, and we create a separate PR for the functionality. Should I create that PR, and if so what exactly should I specify in the PR description?

Reply:
Rereading the section after a long time, I'm mostly confused by the word "tuple." 
AFAIK the line
```
A_multiple = df.loc[:, ['race','gender']]
```
just creates a DataFrame with two columns, rather than 1 column with a tuple as the value, right? I'm guessing that's what you meant and I'm just being pedantic about wording.

If that's true, then I think we already have the best solution and you can ignore what I wrote above as it doesn't really apply.